src/components/Settings/SettingsNotifications/NotificationModal/NtfyModal.tsx

```
@@ -0,0 +1,324 @@
+  onTest: (testData: NotificationAgentNtfy) => void;
+  onSave: (submitData: NotificationAgentNtfy) => void;
+}
+
+const NtfyModal = ({ type, data, onClose, onTest, onSave }: NtfyModalProps) => {
+  const intl = useIntl();
+
+  const NotificationsNtfySchema = Yup.object().shape({
+    url: Yup.string()
+      .when('enabled', {
+        is: true,
+        then: Yup.string()
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)